readme.md

```diff
@@ -19,6 +19,9 @@ $resumator = new Resumator("YOUR_API_KEY");
 // return all Jobs
 $jobs = $resumator->getJobs();
 
+// return all Jobs without caching
+$jobs = $resumator->getJobs();
```

Comment:
@robertark is there something missing from this example method call? I cant tell what's returning jobs without caching